packages/openapi/convert_types.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Usage: ./convert_types.sh <path>
+# Replaces all occurrences of 'Record<string, never>' with 'Record<string, any>' in the given file or recursively in a directory.
```

Comment:
Would replacing with `Record<string, unknown>` work? That would be my preference to capture the unknown nature of the type rather than copy-left effects of any.